ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html

```diff
@@ -17,8 +17,8 @@
       <nav class="hmcts-header__navigation" aria-label="Account navigation">
         <ul class="hmcts-header__navigation-list">
           <li *ngIf="loggedIn" class="hmcts-header__navigation-item">
-            <a id="header-logout-link" class="govuk-header__link  right-align" href="javascript:void(0)"
-              (click)="logout()" i18n="@@header_btn_signout">
+            <a id="header-logout-link" class="govuk-header__link  right-align" href="/logout"
+               i18n="@@header_btn_signout">
```

Comment:
I'm not sure we should use href in an angular app as it reloads the page. It might be fine since it's a logout but I will have to confirm.

Reply:
This could cause issues as the logic is in the logout component. Please revert or use routerLink instead.

Reply:
https://angular.io/api/router/RouterLink

Reply:
A similar change the other way being made in video web: https://github.com/hmcts/vh-video-web/pull/1510/files#r686123961

We should make sure we have a consistent way to do this.

Reply:
The previous javascript function was having an erroneous effect and made it compatible with the usage and pattern followed elsewhere  in Video Web and Admin Web. In that case, we need to standardise this as a technical debt and need to standardise in all the solutions

Reply:
Going forward we will use the routerlink method seen in the PR in my previous comment. We should make that change now rather than waiting.

Reply:
Changes here may break automation, please keep us up to speed so we can change our testing at the earliest opportunity.